sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/LruCache.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections;
```

Comment:
I was going to suggest that we share this from the base SR library, but it's simple and focused enough that I think the duplication is better than having an implicit dependency.

Reply:
Yeah, I also thought about it but since it's one file it seems easier to duplicate for this case.